Changed the transaction in Chapter-1, Challenge-2

shaavan opened the pull request:
Resolves #185 

- This PR replaces the transaction from `75764fd0c95b4c17b728d10f7555509adfc0789ddc47683c45aeddd1c34727f8` to `ee3b8caaeb58245338dd299467de89ec6833d2a4235493c95059934603b5e98d` in both parts of Chapter-1, Challenge-2.
- I have removed the following part from the success message.

```
...We don’t know why tacos were so important to the anonymous sender of this transaction, but here we are.
And it’s an admirable goal at least.
```

- This can later be replaced with a more appropriate success message.

ecurrencyhodler commented:
tACK!

vinayaksh42 reviewed (approved):
great work @shaavan, code-wise everything looks fine. I think we should check with @GBKS regarding the copy message when the challenge completes.

ecurrencyhodler commented:
@shaavan 

Let's update it using the copy here for now. https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=2136%3A24559&t=UIr1v4pegcYfMnU4-1

"That’s correct! Nice work. 

As you can see, the clue is an address. Go to it. 

There you’ll find an abandoned warehouse with all the tools you need to fight BitRey and bring equilibrium back to bitcoin.

Your next challenge awaits you."

We'll need to change the copy for the rest of the chapter but that can wait till it is finalized.

shaavan commented:
Updated from 85ab7f2a64f1a74ee9a6a9dac34fd18c4794805a to 498832f16f54439bd652ee160905373b03b27b5b ([pr223.01](https://github.com/shaavan/saving-satoshi/commits/pr223.01) -> [pr223.02](https://github.com/shaavan/saving-satoshi/commits/pr223.02), [diff](https://github.com/shaavan/saving-satoshi/compare/pr223.01..pr223.02))
Addressed @ecurrencyhodler comment

Changes:

1. Changed the success message to match the [Figma](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=2136%3A24559&t=UIr1v4pegcYfMnU4-1) designs
